src/main/java/com/example/springBoot_hibernate_ferrocarriles/controller/CocheMotorController.java

```diff
@@ -0,0 +1,54 @@
+package com.example.springBoot_hibernate_ferrocarriles.controller;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.example.springBoot_hibernate_ferrocarriles.model.CocheMotor;
+import com.example.springBoot_hibernate_ferrocarriles.service.CocheMotorService;
+
+@RestController
+public class CocheMotorController {
+	@Autowired
+	CocheMotorService cocheMotorService;
+
+	public CocheMotorController() {
+		super();
+	}
+	
+	@GetMapping("/cocheMotor")
```

Comment:
Just a suggestion, URL are not necessarily Java conventions, it is ok to have: `cochemotor`.